### DISCUSSION TITLE
Preconfigured script option in user_config_override.h?

### DISCUSSION THREAD

**Discussion by Arnold-n** [arendst/Tasmota#13149] [Ideas]
For one of my projects I find it convenient to have a script preloaded in the firmware image by
defining it in user_config_override.h (#define PRECONFIGURED_SCRIPT ">D.....") .

In addition, I use #define START_SCRIPT_FROM_BOOT to enable this script by default.

It consists of a few extra lines: https://github.com/Arnold-n/Tasmota/commit/0de562567b134872da517662a46df27ea4f82ad3
 
Would this be of interest to add upstream?

**Comment by barbudor** [comment]
Similar things have been done recently for rules, so that seems to be in the same idea.
Please submit a PR and that will be discussed by the dev team

**Comment by sfromis** [comment]
Implemented in #13153

**Reply by cepela** [reply]
Great, thank you.
